sim/priest/devouring_plague.go

```diff
@@ -47,31 +52,24 @@ func (priest *Priest) registerDevouringPlagueSpell() {
 			ActionID: actionID,
 		}),
 
-		NumberOfTicks: 8,
-		TickLength:    time.Second * priest.ApplyCastSpeed(3),
-
+		NumberOfTicks:       8,
+		TickLength:          time.Second * priest.ApplyCastSpeed(3),
```

Comment:
i think you can use the `AffectedByCastSpeed: true` and then set the tick length to just `time.Second*3`